File: begineer/DisplayOdd.cpp
#include<iostream>
int main(){
	int a,b;
	std::cin>>a>>b;
	for(int i=a+1;i<b;++i){
		if((i&1)==0){
			std::cout<<i<<" ";
		}
	}
	return 0;
}

File: begineer/DisplayOdd.java
import java.util.Scanner;
public class DisplayOdd{
	public static void main(String[] args){
		Scanner cin= new Scanner(System.in);
		int a=cin.nextInt();
		int b=cin.nextInt();
		for(int i=a+1;i<b;++i){
			if((i & 1 )==1){
				System.out.print(i+" ");
			}
		}
	}
}
